libs/documentation/src/lib/components/formly/fileinfo/demos/basic/fileinfo-basic.module.ts

```diff
@@ -0,0 +1,18 @@
+import { FormlyModule } from '@ngx-formly/core';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { FormlyFileInfoBasic } from './fileinfo-basic.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    FormsModule,
+    FormlyModule.forRoot(),
```

Comment:
Could you also add in SdsFormlyModule as import here? It probably doesn't throw an error in our documentation because some other module will load it in, but I was getting errors for it when setting up in isolated stackblitz environment.

Reply:
This is demo is not specific to the filter, it's common for formly type, so I am not using SdsFormlyModule.